app/templates/components/widgets/forms/billing-info.hbs

```diff
@@ -0,0 +1,41 @@
+<div class="field">
```

Comment:
Can we move it back to the template now?

Reply:
It looks more organised since its a just a tool which is needed for the `order-form`, I dont see any issue in moving it to a widget?